cozy_management/cli.py

```diff
@@ -237,6 +241,15 @@ def main():
     if arguments['restore']:
         backup.restore(arguments['<backup_filename>'])
 
+    if arguments['install_weboob']:
+        weboob.install()
+        
+    if arguments['update_weboob']:
```

Comment:
can join in the if above:

`if arguments['install_weboob'] or arguments['update_weboob']`